packages/admin-panel/src/App.js

```diff
@@ -17,7 +17,6 @@ import { labelToId } from './utilities';
 
 export const App = ({ user }) => {
   const headerEl = React.useRef(null);
-  const match = useRouteMatch()
```

Comment:
the match is reading the context of where the hook is placed so the top level App component is always matching `/`